config/interpolate_funcs.go

```diff
@@ -941,6 +942,25 @@ func interpolationFuncReplace() ast.Function {
 	}
 }
 
+// interpolationFuncReverse implements the "reverse" function that does list reversal
+func interpolationFuncReverse() ast.Function {
+	return ast.Function{
+		ArgTypes:   []ast.Type{ast.TypeList},
+		ReturnType: ast.TypeList,
+		Variadic:   false,
+		Callback: func(args []interface{}) (interface{}, error) {
+			inputList := args[0].([]ast.Variable)
+
+			reversedList := make([]ast.Variable, len(inputList))
+			for idx := range inputList {
```

Comment:
One could say that you could iterate over `len(inputList) /2` and do swaps:
example pseudocode
```
        r := make([]int, len(inputList))
	copy(r, inputList)
	for i, j := 0, len(r)-1; i < len(r)/2; i, j = i+1, j-1 {
		r[i], r[j] = r[j], r[i]
	}
	return r, nil
```